### PR TITLE
documentation: ZENKO-1546_Clarify_proxies_in_Installation_Guide

### PR DESCRIPTION
**Pull request by wabernat**
This PR clarifies proxy configuration in the Installation Guide. 

It fixes #ZENKO-1546

### CODE REVIEW

File: docs/Installation/topics/Installing_Zenko.rst [hunk 1]
```diff
@@ -129,7 +129,10 @@ Follow these steps to install Zenko with Ingress.
    “:doc:`configure_ingress`” for additional terms to add to this chart.
 
 3. If your Zenko instance is behind a proxy, add the following lines to the
-   options.yml file, entering your proxy’s IP addresses and port assignments:
+   options.yml file, entering your proxy’s full IP address/URL and port
```
Comment (rahulreddy) [inline docs/Installation/topics/Installing_Zenko.rst]: I think `proxy server address` makes more sense than saying IP/URL/address/locator to be consistent. Something like `entering your proxy's full server address including protocol and port` would work.

File: docs/Installation/topics/Installing_Zenko.rst [hunk 2]
```diff
@@ -139,13 +142,24 @@ Follow these steps to install Zenko with Ingress.
         caCert: false
         no_proxy: ""
 
+   For example: 
+
+   ::
+
+    cloudserver:
+      proxy:
+        http: "http://proxy.example.com"
```
Comment (rahulreddy) [inline docs/Installation/topics/Installing_Zenko.rst]: Let's add port here as well `http://proxy.example.com:80`

**Review by JianqinWang — APPROVED** [review]

**Review by rahulreddy — APPROVED** [review]

**Review by ssalaues — APPROVED** [review]

**Comment by ssalaues** [comment]
@wabernatScality This one just missing your `berte approve` to go through

**Comment by wabernat** [comment]
@bert-e approve

**Pull request merged**